src/codegen/index.js

```diff
@@ -9,8 +10,14 @@ const hasOwnProperty = (obj, prop) =>
   Object.prototype.hasOwnProperty.call(obj, prop)
 
 const isVariable = (expr) => expressions.variable.test(expr)
+
 const getVariableName = (expr) => (expressions.variable.exec(expr) || [])[1]
 
+const isMultiVariableString = (expr) => {
+  const matches = [...expr.matchAll(expressions.variables)]
```

Comment:
Do we have a polyfill for the `matchAll`-function? 

https://caniuse.com/?search=matchAll

Reply:
For node usage we're good (node 12). For cloud app usage, It looks like core-js covers that.

Reply:
Great!

Reply:
just FYI there is no more core-js in the cloud/k6, but k6 has support for [matchAll from goja](https://github.com/dop251/goja/pull/248)

Reply:
@MStoykov The `matchAll` is run in node/browser when converting the har file to a script. So it doesn't run in k6.